pypeit/spec2dobj.py

```diff
@@ -79,6 +79,13 @@ class Spec2DObj(datamodel.DataContainer):
                  'sci_spat_flexure': dict(otype=float,
                                           descr='Shift, in spatial pixels, between this image '
                                                 'and SlitTrace'),
+                 'sci_spec_flexure': dict(otype=np.ndarray, atype=np.floating,
+                                          descr='Global shift of the spectrum to correct for spectral'
+                                                'flexure (pixels). This is based on the sky spectrum at'
+                                                'the center of each slit'),
+                 'vel_type': dict(otype=str, descr='Type of heliocentric correction (if any)'),
```

Comment:
Is there a place where these are listed?  Are can you just provide the list here?

Reply:
Indeed - I've added a link to this, and listed the current options available.